scripts/Genetics.zs

```diff
@@ -204,10 +204,10 @@ Assembler.addRecipe(<Genetics:misc:9>, <Forestry:chipsets:1>,  <gregtech:gt.meta
 Assembler.addRecipe(<Genetics:misc:9>, <Forestry:chipsets:1>,  <gregtech:gt.metaitem.01:32703> * 2, <liquid:molten.stainlesssteel> * 64, 400, 64);
 
 // --- Integrated Casing
-Assembler.addRecipe(<Genetics:misc:11>, <Forestry:hardenedMachine>, <Genetics:misc:9> * 8, <liquid:molten.glowstone> * 288, 1500, 256);
+Assembler.addRecipe(<Genetics:misc:11>, <Forestry:hardenedMachine>, <Genetics:misc:9> * 4, <liquid:molten.glowstone> * 288, 1500, 120);
 
```

Comment:
I don't understand why you want me to put control circuits in recipes that have 0 possibility of recipe conflicts? The only thing it could possibly conflict with is the next recipe, which uses a different first item.

Reply:
Long time ago i wasnt able to add a circuit because assembler had only 2 slots. Now it is possible so i add it to avoid conflicts in the future. Whenever i add a new recipe i will add some